.github/workflows/docker.yml

```diff
@@ -1,19 +1,19 @@
 name: Build Base Docker Image
 
-# Triggered only if environment.yml in main changes
+# Triggered only if environment_meson.yml in main changes
 # Pushes a Docker image with all dev dependencies to Docker Hub
 on:
   push:
     branches:
       - main
     paths:
-      - 'environment.yml'
+      - 'environment_meson.yml'
 
 jobs:
   build:
     name: Build base Docker image
     runs-on: ubuntu-latest
-    environment: scipy-dev
```

Comment:
Same as in `.github/workflows/gitpod.yml` this should not be changed but kept as `scipy-dev`